scripts/run_benchmarks.py

```diff
@@ -309,6 +313,7 @@ def main(args: ArgumentParser):
 
     parser = ArgumentParser()
     parser.add_argument("--cache_dir", type=str, default=None)
+    parser.add_argument("--chain_endpoint", type=str, default="finney")
```

Comment:
Can we reuse the subtensor args? i.e.

 bt.subtensor.add_args(parser)

Then we can provide the arg via --subtensor.chain_endpoint